src/gt4py/utils/meta.py

```diff
@@ -244,6 +245,32 @@ def generic_visit(self, node, **kwargs):
         return node
 
 
+class FullNameCreator(ast.NodeVisitor):
```

Comment:
Let's use the name "qualified name" not "full name". A visitor is fine with me, but I would simply use a function like this:

```python
def get_qualified_name(node: Union[ast.Name, ast.Attribute]):
  if isinstance(node, ast.Name):
    return node.id
  elif isinstance(node: ast.Attribute):
    return node.attr + "." + get_qualified_name(node.value)
  raise ValueError(f"Expected a node of type Union[ast.Name, ast.Attribute], but got {type(node)}")
```

Seems shorter an easier to digest to me.

Reply:
Ah yes, a function is much superior. Thanks for the suggestion! I'll make that change now.